src/components/NavBar.tsx

```diff
@@ -24,11 +24,12 @@ const Logo = config.ui?.Logo;
 
 const useStyles = makeStyles((theme) => ({
   appBar: {
-    position: "fixed",
+    position: "relative",
```

Comment:
```suggestion
    position: "sticky",
```
This should have the same sticky behaviour as before (which I like), but without the need to add the fixed 64px padding on top.